README.md

```diff
@@ -1,8 +1,11 @@
-<img src="https://s3.amazonaws.com/aws-mobile-hub-images/aws-amplify-logo.png" alt="AWS Amplify" width="225">
+<img src="https://s3.amazonaws.com/aws-mobile-hub-images/aws-amplify-logo.png" alt="AWS Amplify Logo" width="225">
```

Comment:
Not sure if it really makes a difference, but can we use the Amplify Logo in `./docs/public/amplify-logo.svg`?

Reply:
Those 2 are different. ./docs/public/amplify-logo.svg only has the logo, without text